docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,51 @@
+# Stepback Bisection
+Evergreen's stepback bisection performs stepback by continuously reducing the amount of commits needed to test by half. To enable it, go to the project settings 'general' tab.
+
+## Workflow
+Stepback is Evergreen's workflow for finding a commit in a chain of untested commits that breaks a test. A chain of untested commits happens when many commits are pushed upon the tracking branch at once so a patch is only ran for the HEAD commit (patches are made for the commits between the HEAD and the previous HEAD, but they are unscheduled). If a test fails, Evergreen performs stepback (if stepback is enabled in the project configuration file / project settings).
+
+## Strategy
+Traditionally, Evergreen performed linear stepback only. This is testing each previous commit one by one. If stepback bisection is enabled, Evergreen with use bisection instead of linear.
+
+Stepback bisection tests the commit between the HEAD and last passing commit, effectively breaking the searchable area in half. If there is no clean mid commit, it will take the earlier commit. It continues this until we have a failing and passing commit side-by-side (see the example below).
```

Comment:
The task that triggers stepback doesn't strictly have to be the HEAD, so we should change this to say something more generic like "the failing mainline commit"